src/test/java/com/celements/model/util/ClassFieldValueTest.java

```diff
@@ -18,14 +18,14 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.